resources/qml/ProfileAndSettingComponent.qml

```diff
@@ -0,0 +1,241 @@
+            }
+
+            background: Rectangle
+            {
+                border.width: control.checked ? UM.Theme.getSize("default_lining").width * 2 : UM.Theme.getSize("default_lining").width
+                border.color: (control.checked || control.pressed) ? UM.Theme.getColor("action_button_active_border") : control.hovered ? UM.Theme.getColor("action_button_hovered_border"): UM.Theme.getColor("action_button_border")
+
+                // for some reason, QtQuick decided to use the color of the background property as text color for the contentItem, so here it is
+                color: (control.checked || control.pressed) ? UM.Theme.getColor("action_button_active") : control.hovered ? UM.Theme.getColor("action_button_hovered") : UM.Theme.getColor("action_button")
+            }
+
+            contentItem: Label
+            {
+                text: model.text
+                font: UM.Theme.getFont("default")
+                horizontalAlignment: Text.AlignHCenter
+                verticalAlignment: Text.AlignVCenter
+                renderType: Text.NativeRendering
+                elide: Text.ElideRight
+                color:
+                {
+                    if(control.pressed)
+                    {
+                        return UM.Theme.getColor("action_button_active_text");
+                    }
+                    else if(control.hovered)
+                    {
+                        return UM.Theme.getColor("action_button_hovered_text");
+                    }
+                    return UM.Theme.getColor("action_button_text");
+                }
+            }
+        }
+    }
+
+    Item
+    {
+        id: sidebarContents
+        anchors.top: settingsModeSelection.bottom
+        anchors.topMargin: UM.Theme.getSize("sidebar_margin").height
+        anchors.left: parent.left
+        anchors.right: parent.right
+        height: 500
+
+        // We load both of them at once (instead of using a loader) because the advanced sidebar can take
+        // quite some time to load. So in this case we sacrifice memory for speed.
+        SidebarAdvanced
+        {
+            anchors.fill: parent
+            visible: currentModeIndex == 1
+            onShowTooltip: base.showTooltip(item, location, text)
+            onHideTooltip: base.hideTooltip()
+        }
+
+        SidebarSimple
+        {
+            anchors.fill: parent
+            visible: currentModeIndex != 1
+            onShowTooltip: base.showTooltip(item, location, text)
+            onHideTooltip: base.hideTooltip()
+        }
+    }
+
+    // Setting mode: Recommended or Custom
+    ListModel
+    {
+        id: modesListModel
+    }
+
+    Component.onCompleted:
+    {
+        modesListModel.append({
+            text: catalog.i18nc("@title:tab", "Recommended"),
+            tooltipText: catalog.i18nc("@tooltip", "<b>Recommended Print Setup</b><br/><br/>Print with the recommended settings for the selected printer, material and quality.")
```

Comment:
Even better: 

```
tooltipText: "<b>%1</b><br/><br/>%2".arg(catalog.i18nc("@tooltip:title", "Recommended Print Setup")).arg(catalog.i18nc("@tooltip", "Print with the recommended settings for the selected printer, material and quality."))
```

Since that avoids costly string concatenations and makes it easier to see the HTML.

Reply:
Done in 3eb71021e24b7af44626e5b4ffe1a91915f88072